bin/publish

```diff
@@ -7,45 +7,59 @@ set -e
 readonly REGISTRY="cyberark"
 readonly VERSION="$(short_version_tag)"
 readonly FULL_VERSION_TAG="$(full_version_tag)"
-readonly INTERNAL_REGISTRY="registry.tld"
 readonly REDHAT_IMAGE="scan.connect.redhat.com/ospid-18d9f51d-9c0c-4031-9f9e-ef08aa2ff409/secretless-broker"
 readonly IMAGES=(
   "secretless-broker"
   "secretless-broker-quickstart"
 )
 
-readonly TAGS=(
-  "$VERSION"
-  "latest"
-)
+function main() {
+  TAG_NAME="${TAG_NAME:-}"
 
```

Comment:
should we also have a check here for if TAG_NAME is not set, and have the script fail in this case? this script should only be run from tag-triggered builds, I think

similar to here: https://github.com/cyberark/parse-a-changelog/blob/69f941471eca1a24081f573959875ce4ff54bbd3/publish.sh#L5:L10